scss-test.scss

```diff
@@ -0,0 +1,19 @@
+.unsorted {
+  color: #FFFFFF;
+  background: red;
+  .inception-1 {
+    color: #DDD;
+    .inception-2 {
+       color: #ccc;
+       inception-3 {
+         color: #bbb;
+          inception-4 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5